sessionctx/variable/tidb_vars.go

```diff
@@ -684,6 +684,11 @@ const (
 
 	// TiDBSimplifiedMetrics controls whether to unregister some unused metrics.
 	TiDBSimplifiedMetrics = "tidb_simplified_metrics"
+
+	// TiDBExtensionNonMySQLCompatible changes output for SHOW CREATE TABLE etc. for TiDB enhancements
+	// that are not compatible with MySQL, like CLUSTERED index etc.
+	// as well as remove MySQL specific syntax that are not used by TiDB, like ENGINE = InnoDB
+	TiDBExtensionNonMySQLCompatible = "tidb_extension_non_mysql_compatible"
```

Comment:
For bool sysvars, most use the convention "tidb_enable_XXX". Is it possible to change to something of this variety. Maybe `tidb_enable_create_table_extensions`?

Reply:
I just changed it to `tidb_enable_simplified_show_create_table`